front/src/main/ui/pages/Home/SecondPage/CeoMessSignature/CeoMessSignature.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import s from './CeoMessSignature.module.css';
+import signature from '../../../../../../assets/img/home/secondPage/signature.svg'
```

Comment:
add please some 'alias' package to not use so long path

Reply:
done(I didn't quite understand what you meant, therefore look at svgWrapper in UI path) I'm right?